product/views.py

```diff
@@ -1,3 +1,90 @@
-from django.shortcuts import render
+from django.views   import View
+from django.http    import JsonResponse
 
-# Create your views here.
+from .models        import MainCategory, Product, ProductGroup
+
+class CategoryView(View):
+    def get(self, request):
+        categories = MainCategory.objects.all()
+        category_list = [{
+            "id"   : category.id,
+            "name" : category.name
+        } for category in categories]
+
+        return JsonResponse({"category": category_list}, status=200)
```

Comment:
변수명을 정할때, 복수 단수도 맞춰주세요~
```suggestion
        return JsonResponse({"category": category_list}, status=200)
```

Reply:
수정했습니다~